_docs/2.0.0/usage/standardization-formats.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: docs
+title: Usage - Standardization Formats
+version: '2.0.0'
+categories:
+    - '2.0.0'
+    - usage
+---
+
+Currently Standardization supports these formats
+
+- [Cobol](#cobol) (see [Cobrix GitHub](https://github.com/AbsaOSS/cobrix/))
+- [CSV](#csv) (see [rfc4180](https://tools.ietf.org/html/rfc4180))
+- [FixedWidth](#fixed-width) (see `Link to be added repo does not exist yet`)
+- [JSON](json) (see [json.org](https://www.json.org/json-en.html))
+- [Parquet](parquet) (see [Apache Parquet](https://parquet.apache.org/documentation/latest/))
```

Comment:
Dont you mean `#json` & `#parquet` (in-file links?)

Reply:
Good catch